src/modules/taxonomy/taxonomyApi.js

```diff
@@ -42,6 +42,12 @@ function fetchRelevances(locale) {
   );
 }
 
+function fetchSubject(subjectId) {
+  return fetchAuthorized(`${baseUrl}/subjects/urn:subject:${subjectId}`).then(
```

Comment:
subjectId bør overalt være heile urn-en og ikkje berre tallet. 
```suggestion
  return fetchAuthorized(`${baseUrl}/subjects/${subjectId}`).then(
```